doc/installing.rst

```diff
@@ -38,36 +38,153 @@ System Requirements
 .. note::
    For Linux distributions, we support, if technically feasible, the latest 2 versions per platform and the previous Ubuntu `LTS`_.
 
+Customizing the Windows installation
+----------------------------------
```

Comment:
This will cause a render error, as the underline has to be the same length as the text above. Actually, it might be, but the GitHub font makes it appear as though it isn't.